### ISSUE TITLE
[Bug] Upgrade sql script may cause error when user database is not streampak.

### ISSUE THREAD

**Issue by SbloodyS** [apache/streampark#3400]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### Java Version

1.8

### Scala Version

2.11.x

### StreamPark Version

2.1.2

### Flink Version

1.17.1

### deploy mode

None

### What happened

above.

### Screenshots

![a257xn4SzIfFvVC.png](https://cdn.sa.net/2023/12/15/a257xn4SzIfFvVC.png)

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!(您是否要贡献这个PR?)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by hackallan** [comment]
Change the database name to your own database name, I don't think this is a bug.

**Comment by SbloodyS** [comment]
> Change the database name to your own database name.

Yes. For developers, this can be done, but for users, it may not be as familiar. There are no relevant upgrade instructions in the documentation, which can lead to upgrade failure, so I think this is a bug.

**Comment by caicancai** [comment]
Good catch

**Comment by wolfboys** [comment]
Thanks for your feedback. We are closing this issue because it is outdated.
